common/pb/CMakeLists.txt

```diff
@@ -23,7 +23,7 @@ SET(PROTO_FILES
     command_delete_arrow.proto
     command_draw_cards.proto
     command_dump_zone.proto
-    command_flip_card.proto
+    command_turn_card_over.proto
```

Comment:
Don't change the proto

Reply:
So do we keep track of such left out changes and do them at one point combined?

In addition to requesting the server admins to require that "feature" when connecting to their servers?
Or bumping Cockatrice to v3 as incompatible change?

Reply:
Add a comment here, but otherwise leave all the proto files alone.

This should only be a client-facing string change.